src/coreclr/src/inc/corinfo.h

```diff
@@ -1321,6 +1321,7 @@ struct CORINFO_LOOKUP_KIND
 //
 #define CORINFO_MAXINDIRECTIONS 4
 #define CORINFO_USEHELPER ((WORD) 0xffff)
+#define CORINFO_SKIPSIZECHECK ((WORD) 0xffff)
```

Comment:
Nit: A better name may be `CORINFO_NO_SIZE_CHECK`. 

The names on JIT/EE tend to use NO prefix to describe that something should not happen, e.g `CORINFO_NO_THIS_TRANSFORM`, `CORJIT_FLAG_NO_INLINING`, ...

Reply:
Renamed